src/main/java/uk/gov/hmcts/darts/usermanagement/service/impl/UserManagementServiceImpl.java

```diff
@@ -135,6 +135,11 @@ private UserAccountEntity updatedUserAccount(UserPatch userPatch, UserAccountEnt
     }
 
     private void updateEntity(UserPatch userPatch, UserAccountEntity userAccountEntity) {
+        if (userPatch.getEmailAddress() != null && !userPatch.getEmailAddress().equals(userAccountEntity.getEmailAddress())) {
```

Comment:
you can use Apache commons StringUtils.equals which will handle the nulls automatically and make the code slightly cleaner.